public/docs/ts/latest/guide/architecture.jade

```diff
@@ -193,196 +224,208 @@ figure
 figure
   img(src="/resources/images/devguide/architecture/component-tree.png" alt="Metadata" align="left" style="width:300px; margin-left:-40px;margin-right:10px" )
 :marked
-  Notice how `<hero-detail>` rests comfortably among the HTML elements we already know. 
-  We can mix ... and will mix ... our custom components with native HTML in the same layouts.
-  
-  And in this manner we can and will compose complex component trees to build out our richly featured application.
+  Notice how `<hero-detail>` rests comfortably among native HTML elements.
+  We can and _will_ mix our custom components with native HTML in the same layouts.
+
+  In this manner we'll compose complex component trees to build out our richly featured application.
 <br clear="all">
 
 .l-main-section
-<a id="metadata"></a>
 :marked
-  ## Angular Metadata
+  ## Metadata
 figure
   img(src="/resources/images/devguide/architecture/metadata.png" alt="Metadata" align="left" style="width:150px; margin-left:-40px;margin-right:10px" )
+
 :marked
 <p style="padding-top:10px">Metadata tells Angular how to process a class.</p>
 <br clear="all">
 :marked
-  [Looking back](#component-code) at the `HeroListComponent`, we see that it's just a class.
+  [Looking back at the code](#component-code) for `HeroListComponent`, we see that it's just a class.
   There is no evidence of a framework, no "Angular" in it at all.
-  
+
   In fact, it really is *just a class*. It's not a component until we *tell Angular about it*.
-  
+
   We tell Angular that `HeroListComponent` is a component by attaching **metadata** to the class.
-  
-  The easy way to attach metadata in TypeScript is with a **decorator**. 
+
+  In !{_Lang}, we attach metadata by using an **!{_decorator}**.
```

Comment:
Is this going to render as "an decorator" in TS? I thought you had another token that distinguished "a decorator" (ts) from "an annotation" (dart).